app/assets/javascripts/services/docCacheSvc.js

```diff
@@ -55,7 +55,7 @@ angular.module('QuepidApp')
         var docIds    = Object.keys(docsToFetch);
         var resolver  = docResolverSvc.createResolver(docIds, settings, 15);
 
-        if ( docIds.length > 0 ) {
+        if ( docIds.length > 0 && settings.searchEngine !== 'vectara') { // 'vectara' does not support doc lookup by ID
```

Comment:
This is totally fine...  And pionts to, in the future, we need some sort of "capablities" so you can be like `settings.searchEngine.supportDocLookup` instead ;-)